src/rgw/rgw_rest_s3.cc

```diff
@@ -246,7 +257,7 @@ int RGWGetObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t bl_ofs,
 
   if (! op_ret) {
     if (! lo_etag.empty()) {
-      /* Handle etag of Swift API's large objects (DLO/SLO). It's entirerly
+      /* Handle etag of Swift API's large objects (DLO/SLO). It's entirely
```

Comment:
As per coding guidelines, Please use "It is" instead of It's.
This may not be your change, but yes good to do.
A PR is already raised/accepted for the same.